eclass/xdg.eclass

```diff
@@ -51,24 +52,35 @@ xdg_pkg_preinst() {
 		XDG_ECLASS_DESKTOPFILES+=( ${f} )
 	done < <(cd "${D}" && find 'usr/share/applications' -type f -print0 2>/dev/null)
 
+	XDG_ECLASS_ICONFILES=()
+	while IFS= read -r '' f; do
+		XDG_ECLASS_ICONFILES+=( ${f} )
+	done < <(cd "${D}" && find 'usr/share/icons' -type f -print0 2>/dev/null)
```

Comment:
"${ED}" please.

Reply:
doesn't D vs ED technically just have to go into deeper subdirectories from D vs already a deeper subdir with ED? But given we updates the cache found in $EROOT, this correction makes sense

Reply:
I don't think find recurses to find the inital path you pass to it. If EPREFIX is non-empty, I would expect the original version of this find call to find nothing.